common/templates/start/_adoc-file

```diff
@@ -2,81 +2,117 @@
+// 1. Define and use document attributes or variables in this file.
+// 2. Update the docinfo.xml file if needed.
 //
 // = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
 
+
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-// organization - do NOT modify
+// ORGANIZATION
+//   Do NOT modify this section.
 // -
 :trd: Technical Reference Documentation
 :type: Getting Started
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 
+
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// DOCUMENT REVISION DATE
+//-
+:revision-date: YYYY-MM-DD
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// DOCUMENT TITLE AND SUBTITLE
+//   Provide concise but descriptive title and subtitle.
+// -
+:title: (<75 characters) Concisely identify the guide (e.g., "Kubeflow with Rancher")
+:subtitle: (<75 characters) Use for clarification (e.g., "Deploying Kubeflow onto an RKE2 Cluster with Rancher")
+
+
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// TECHNICAL COMPONENTS
+//   Identify the technical components featured in the guide.
+//   Define variables for various product information, such as
+//     product names (long and short), versions, websites, etc.
+//   For additional components, replicate the pattern (e.g.,
+//     scomp2, scomp3-version, pcomp2-website, etc.)
+//   Remove or comment out unneeded variables.
+// -
+// SUSE Components
+// -
+:scomp1-provider: SUSE
+:scomp1: Primary SUSE product short name (e.g., SLES, Rancher Prime)
+:scomp1-full: Primary SUSE product full name (e.g., SUSE Linux Enterprise Server, Rancher Prime by SUSE)
+:scomp1-version: Primary SUSE product relevant version (e.g., '15', '15SP5', '2.7') or versions (e.g., '15SP4, 15SP5', '15SP3+', '2.6+', '2.x')
+:scomp1-website: Primary SUSE product website URL
+:scomp1-docs: Primary SUSE product documentation URL
+
+// -
+// Partner/Project Components
+// -
+:pcomp1-provider: Partner/Project name
+:pcomp1: Partner/Project component short name
+:pcomp1-full: Partner/Project component full name
+:pcomp1-version: Partner/Project component version or versions
+:pcomp1-page: Partner/Project component website URL
+:pcomp1-docs: Partner/Project component documentation URL
+
+
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-// document
+// DOCUMENT DESCRIPTIONS
+//   Provide information to identify the guide by its purpose,
+//     use case, value.
 // -
-:title: Title
-:subtitle: Subtitle
+:usecase: (<55 characters) (e.g., Database-as-a-Service, edge analytics in healthcare, Kubernetes-native object storage)
+
+:executive-summary: Give a brief summary of what this document provides (e.g., "Streamline deployment and management of multi-tenant Kubernetes landscapes with {pcomp1-full} and {scomp1-full}.")
```

Comment:
As I found out, this goes into an `abstract/para` element. Not sure if we should add a hint about how many characters are useful. As this is not really a metadata (it's free text) there are no limits.

I just mention it here as I stumbled upon this and thought "hmn, are there no character limits?". :wink: I was first a bit confused as the other variables contains this note.

Reply:
Yes, that is a bit hard to read.
I should move the instructions to a free-standing comment, perhaps.